README.md

```diff
@@ -56,15 +56,7 @@ sudo sysctl -w vm.max_map_count=262144
 ```
 
 
-Now you need to build the required docker containers. Fortunately, the gradle build file can do this for you.
-
-From the CogStack top level directory:
-
-```
-gradlew buildSimpleContainers
-```
-
-Assuming the containers have been built successfully, simply navigate to
+Now simply navigate to
 ```
 cd docker-cogstack/compose-ymls/simple/
```

Comment:
don't think all these containers are cloud built yet

Reply:
Yes but i have specified build context in the docker-compose.yml file so should automatically build when before running